streampipes-processors-filters-jvm/src/main/java/org/apache/streampipes/processors/filters/jvm/processor/schema/MergeBySchemaProcessor.java

```diff
@@ -0,0 +1,60 @@
+package org.apache.streampipes.processors.filters.jvm.processor.schema;
+
+import org.apache.streampipes.commons.exceptions.SpRuntimeException;
+import org.apache.streampipes.model.DataProcessorType;
+import org.apache.streampipes.model.graph.DataProcessorDescription;
+import org.apache.streampipes.model.runtime.Event;
+import org.apache.streampipes.model.schema.EventSchema;
+import org.apache.streampipes.sdk.builder.ProcessingElementBuilder;
+import org.apache.streampipes.sdk.builder.StreamRequirementsBuilder;
+import org.apache.streampipes.sdk.helpers.EpRequirements;
+import org.apache.streampipes.sdk.helpers.Locales;
+import org.apache.streampipes.sdk.helpers.OutputStrategies;
+import org.apache.streampipes.sdk.utils.Assets;
+import org.apache.streampipes.wrapper.context.EventProcessorRuntimeContext;
+import org.apache.streampipes.wrapper.routing.SpOutputCollector;
+import org.apache.streampipes.wrapper.standalone.ProcessorParams;
+import org.apache.streampipes.wrapper.standalone.StreamPipesDataProcessor;
+
+
+public class MergeBySchemaProcessor extends StreamPipesDataProcessor {
+
+
+    @Override
+    public DataProcessorDescription declareModel() {
+        return ProcessingElementBuilder.create("org.apache.streampipes.processors.filters.jvm.schema")
+                .category(DataProcessorType.TRANSFORM)
+                .withAssets(Assets.DOCUMENTATION, Assets.ICON)
+                .withLocales(Locales.EN)
+                .outputStrategy(OutputStrategies.custom(true))
```

Comment:
```suggestion
                .outputStrategy(OutputStrategies.keep())
```
We do not want to change the events and simply keep them as they are.